osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/stub/DlmsConnectionStub.java

```diff
@@ -146,4 +199,13 @@ public boolean isCloseCalled() {
   private String getKey(final AttributeAddress attributeAddress) {
     return attributeAddress.getInstanceId() + "-" + attributeAddress.getId();
   }
+
+  public void addMethodResul(
```

Comment:
Resul -> Result

Is this method actually used somewhere?

Reply:
No it is not used, removed method